### PR TITLE
edited readme

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

supports; <https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-810>

Make the application readme easier to navigate. 
Include a link to the API docs

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-9f0fbe-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-email-service-pr-42

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-9f0fbe-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-email-service-pr-42

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-9f0fbe-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-email-service-pr-42

File: README.md [hunk 1]
```diff
@@ -26,13 +26,14 @@ To learn more about the **Common Services** available visit the [Common Services
 ## Documentation
 
 * [Application Readme](app/README.md)
+* [API specification](app/README.md#api-usage)
```
Comment (jujaga) [inline README.md]: Capitalize S

File: README.md [hunk 2]
```diff
@@ -26,13 +26,14 @@ To learn more about the **Common Services** available visit the [Common Services
...
 * [Openshift Readme](openshift/README.md)
 * [Devops Tools Setup](https://github.com/bcgov/nr-showcase-devops-tools)
 * [Showcase Team Roadmap](https://github.com/bcgov/nr-get-token/wiki/Product-Roadmap)
 
 ## Getting Help or Reporting an Issue
 
-To report bugs/issues/features requests, please file an issue.
+To report bugs/issues/features requests, please file a Github issue.
```
Comment (jujaga) [inline README.md]: Don't really think this change is needed as this is already on Github

**Pull request merged**